docs/development.md

```diff
@@ -288,5 +288,8 @@ In order create a signature `hello_world.sign` for a Wasm module
 `hello_world.wasm` use the following script:
```

Comment:
(for a future PR)

This text talks about how to generate a signature of a module, but I don't think there's any explanation of *why* you might want such a thing.   Could we add an explanation somewhere (probably in concepts.md, possibly adapted from your helpful [explanation on Slack](https://project-oak.slack.com/archives/CHE9E13C3/p1598369605051800)) of what signature labels mean, then reference it from here?

Reply:
I have a partial explanation here:
https://github.com/project-oak/oak/pull/1396

Not sure if it's enough